vlm/src/main/scala/geotrellis/contrib/vlm/RasterRegion.scala

```diff
@@ -53,4 +53,5 @@ case class RasterRegion(
   def cellType: CellType = source.cellType
   def rasterExtent: RasterExtent = RasterExtent(extent, cols, rows)
   def projectedExtent: ProjectedExtent = ProjectedExtent(extent, crs)
+  def close = source.close
```

Comment:
This doesn't seem right. There could be multiple `RasterRegions` that are floating and have lifetime independent of `RasterSource`. Closing the source here seems like it would be problematic, is there a reason this ended up here?

Reply:
Smth that should not be there; legacy from lots of test iterations.